core/views.py

```diff
@@ -26,32 +28,37 @@ def get_context_data(self, *args, **kwargs):
         )
 
 
-class LandingPageView(TemplateView):
+class SetEtagMixin:
+    def dispatch(self, request, *args, **kwargs):
+        response = super().dispatch(request, *args, **kwargs)
+        if request.method == 'GET':
+            response.add_post_render_callback(set_response_etag)
+        return response
+
+
+class LandingPageView(SetEtagMixin, TemplateView):
     template_name = 'core/landing-page.html'
 
     def get_context_data(self, *args, **kwargs):
-        answer_manager = TriageAnswersManager(self.request)
-        has_completed_triage = answer_manager.retrieve_answers() != {}
         return super().get_context_data(
             *args, **kwargs,
-            has_completed_triage=has_completed_triage,
+            TRIAGE_COMPLETED_COOKIE_NAME=settings.TRIAGE_COMPLETED_COOKIE_NAME,
             casestudies=[
                 casestudies.MARKETPLACE,
                 casestudies.HELLO_BABY,
                 casestudies.YORK,
             ],
-            article_group_read_progress=(
-                self.request.article_read_manager.get_group_read_progress()
-            ),
```

Comment:
removed "3/5 completed" from the landing page

![image](https://user-images.githubusercontent.com/5485798/33176031-8a23e8a0-d055-11e7-8b37-66f230b7c8f6.png)

we can add a ticket to port this behaviour over to javascript.